NEventSocket/Sockets/Parser.cs

```diff
@@ -108,11 +108,11 @@ public Parser Append(char next)
         /// <summary>
         /// Appends the provided string to the internal buffer.
         /// </summary>
-        public Parser Append(string next)
+        public Parser Append(byte[] next)
         {
             var parser = this;
 
-            foreach (var c in next)
+            foreach (var b in next)
             {
                 parser = parser.Append(next);
```

Comment:
Sidenote: this recurses, should be `parser.Append(c)` or now `parser.Append(b)`. This had no effect so far, because `Append(byte[])` is not actually used.